fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/ContentObject.java

```diff
@@ -28,18 +35,35 @@
 
 	protected ContentObject(PID pid, Repository repository, RepositoryObjectDataLoader dataLoader) {
 		super(pid, repository, dataLoader);
-		// TODO Auto-generated constructor stub
 	}
 
 	public void addChild(ContentObject child) {
 		// TODO
 	}
-	
+
 	public BinaryObject addDescription(InputStream contentStream) {
 		return null;
 	}
-	
+
 	public BinaryObject getDescription() {
 		return null;
 	}
+
+	/**
+	 * Retrieve a list of member content objects for this object.
+	 * 
+	 * @return
+	 */
+	public List<ContentObject> getMembers() {
+		List<ContentObject> members = new ArrayList<>();
+		Resource resc = getResource();
+
+		for (StmtIterator it = resc.listProperties(PcdmModels.hasMember); it.hasNext(); ) {
```

Comment:
No, it’s fine to keep it as is. I see some value in the argument for limiting the scope of the iterator. I guess I don’t like the fact that the update happens inside the body of the loop rather than in the header, where I would expect it to be in a for-loop.